automatic/composer/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+
+$fileName = 'Composer-Setup.5.1.0.exe'
+
+$packageArgs = @{
+  packageName  = 'composer'
+  fileType     = 'exe'
+  file         = Get-Item $toolsPath\$fileName
+  checksum     = '80422FEF0EA310C0F565D6028F34010EEEBC96A4EDEC3175156D85F4F0EEEF10'
+  checksumType = 'sha256'
+  silentArgs   = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART'
+  softwareName = 'composer*'
+}
```

Comment:
missing `validExitCodes` argument.
At a minimum, this should be `validExitCodel = @(0)`

Reply:
Done